scripts/setup-cocoapods.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+source_root="$(dirname "$0")/.."
+readonly source_root
```

Comment:
Same comment as above, does the `readonly` prefix need to be on a new line?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2155

Shellcheck complains about combining them because it results in ignoring failures in the subshell.